src/upkeep.rs

```diff
@@ -281,6 +281,7 @@ mod tests {
             on_attempts_exceeded: OnAttemptsExceeded::Discard as i32,
             at_most_once: None,
         });
+        records[1].added_at = records[1].added_at.add(Duration::from_secs(1));
```

Comment:
```suggestion
        records[1].added_at = records[1].added_at + Duration::from_secs(1);
```

Then you don't need to import the `std::ops::Add`.